docs/UserGuide.md

```diff
@@ -212,16 +212,18 @@ Examples:
 
 Sorts the client list in ascending order by the specified field.
 
-Format: sort `FIELD`
+Format: sort `FIELD ORDER`
 
-* `FIELD` can be either `deadline` or `name`.
-* `FIELD` values are case-sensitive; for example, sort name is valid, but sort NAME will result in an error.
-* Sorting by `name` arranges clients in alphabetical order.
-* Sorting by `deadline` arranges clients in ascending order based on their project deadlines, with earlier dates appearing first and later dates at the back.
+* `FIELD` can be either `deadline` or `name`
+* `ORDER` can be either `ascending` or `descending`
 
 Examples:
-* `sort name` sorts the client list alphabetically by name.
-* `sort deadline` sorts the client list by project deadline (earlier dates first).
+* `sort name ascending` sorts the client list alphabetically by name in ascending order.
+* `sort deadline descending` sorts the client list by project deadline (later dates first).
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Addressbook saves you the hassle of sorting the list every time you open the app, so everytime you do a sort, it will save the addressbook
```

Comment:
```suggestion
Clientele+ saves you the hassle of sorting the client list every time you open the app, so every time you do a sort, it will save the list.
```